### PR TITLE
zh-trans: add pull-image-private-registry.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: add pull-image-private-registry.md

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+* `<your-name>` is your Docker username.
+* `<your-pword>` is your Docker password.
+* `<your-email>` is your Docker email.
+
+You have successfully set your Docker credentials in the cluster as a Secret called `regcred`.
+-->
+
+在这里：
+
+* `<your-registry-server>` 是你的私有 Docker Registry FQDN. (参考 https://index.docker.io/v1/ 中关于 DockerHub 的部分)
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Docker Registry -> Docker 仓库

Comment (AdamDang) [reply content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md]: Done！

File: content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+{{% capture steps %}}
+
+<!--
+## Log in to Docker
+
+On your laptop, you must authenticate with a registry in order to pull a private image:
+-->
+
+## 登录 Docker
```
Comment (chenrui333) [inline content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md]: 这里的英文 感觉也有问题，应该是 `登录 Docker Registry`

File: content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+
+```json
+{"auths":{"yourprivateregistry.com":{"username":"janedoe","password":"xxxxxxxxxxx","email":"jdoe@example.com","auth":"c3R...zE2"}}}
+```
+
+<!--
+To understand what is in the `auth` field, convert the base64-encoded data to a readable format:
+-->
+
+要了解 `auth` 字段中的内容，请将 base64 编码的数据转换为可读格式：
```
Comment (chenrui333) [inline content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md]:
```suggestion
要了解 `auth` 字段中的内容，请将 base64 编码过的数据转换为可读格式：
```

File: content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+
+You have successfully set your Docker credentials in the cluster as a Secret called `regcred`.
+-->
+
+在这里：
+
+* `<your-registry-server>` 是你的私有 Docker 仓库 FQDN. (参考 https://index.docker.io/v1/ 中关于 DockerHub 的部分)
+* `<your-name>` 是你的 Docker 用户名。
+* `<your-pword>` 是你的 Docker 密码。
+* `<your-email>` 是你的 Docker email。
```
Comment (chenrui333) [inline content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md]:
```suggestion
 * `<your-email>` 是你的 Docker 邮箱。
```

File: content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+* `<your-name>` is your Docker username.
+* `<your-pword>` is your Docker password.
+* `<your-email>` is your Docker email.
+
+You have successfully set your Docker credentials in the cluster as a Secret called `regcred`.
+-->
+
+在这里：
+
+* `<your-registry-server>` 是你的私有 Docker 仓库 FQDN. (参考 https://index.docker.io/v1/ 中关于 DockerHub 的部分)
```
Comment (chenrui333) [inline content/zh/docs/tasks/configure-pod-container/pull-image-private-registry.md]:
```suggestion
* `<your-registry-server>` 是你的私有 Docker 仓库全限定域名。 (参考 https://index.docker.io/v1/ 中关于 DockerHub 的部分)
```

**Comment by chenrui333** [comment]
add few suggestions, lgtm in general. Thanks!

**Comment by chenrui333** [comment]
Thanks a lot for the updates.

/lgtm
/approve

**Pull request merged**